test/e2e/formats.test.ts

```diff
@@ -85,7 +85,7 @@ describe('push and pull with different formats', () => {
     pak = await createPak(client);
   });
   afterEach(async () => {
-    deleteProject(client);
+    if (client) await deleteProject(client);
```

Comment:
Couldn't this be solved globally in the `deleteProject` method? Feels a bit repetitive